doc/index.rst

```diff
@@ -40,7 +40,7 @@ Installing the First Node
 
 #. Using the ip address that you discovered in the step immediately prior to this step, run the following command::
 
-   $ sudo ./cephadm bootstrap --mon-ip 192.168.1.101 --output-config ceph.conf --output-keyring ceph.keyring --output-pub-ssh-key ceph.pub 
+   $ sudo ./cephadm bootstrap --mon-ip 192.168.1.101 --output-pub-ssh-key ceph.pub 
```

Comment:
we can leave off the output pub ssh key option too-- that also defaults to 'ceph.pub'.